src/main/webapp/script.js

```diff
@@ -377,5 +391,14 @@ function getSchoolIdFromUrl() {
   const id = urlParams.get('school-id');
   return id;
 }
+/**
+ * Redirects home search to college info page after storing user input.
+ */
+function storeAndRedirect() {
+  sessionStorage.clear();
+  const schoolList = document.getElementById('school-search').value;
+  sessionStorage.setItem('schoolList', schoolList);
+  window.location.href = 'college-info.html';
```

Comment:
From our conversation before, I think this is meant to go to search-results.html, is that correct? You're storing the list of schools that came back from the user's search, then you'll display that full list on the search results page.

Reply:
When I can get Toba's edits merged, we had discussed the idea of redirecting to the college-info.html page if there's only one search result that the client's input applies to. I believe this will make it easier when I create the drop-down function post-MVP

Reply:
Yes! For whatever reason yesterday Toba's edits were not pulling. I looked at Toba's code and the redirect works now in my new PR. Thanks